src/runtime/virtcontainers/clh.go

```diff
@@ -525,7 +534,10 @@ func (clh *cloudHypervisor) CreateVM(ctx context.Context, id string, network Net
 	// Followed by extra kernel parameters defined in the configuration file
 	params = append(params, clh.config.KernelParams...)
 
-	clh.vmconfig.Payload.SetCmdline(kernelParamsToString(params))
+	// Set kernel cmdline if no IGVM file set
```

Comment:
May leave a comment that kernel cmdline is encapsulated within IGVM

Reply:
comment updated